m4_sessions/1811_inning/failure.md

```diff
@@ -25,6 +29,26 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Introductions, name, mod, something you are good at and how long it took you to actually become good at it 5 mins
+
+Short video  talking about how failure is a good thing 5 mins
+
+Redefinition of failure 7 mins. (ask how people would define failure, ask how people feel when they fail, ask what they think of others when they fail)
+
+Short video of successful people and their failures 5 min
```

Comment:
Link once you have it!

Reply:
https://www.youtube.com/watch?v=zLYECIjmnQs this is the short video of successful people and their failures